rpmbuild/copr-rpmbuild.spec

```diff
@@ -153,12 +161,13 @@ Requires: yum-utils
 %endif
 %latest_requires mock
 %latest_requires mock-core-configs
+%if !0%{?openEuler}
 %latest_requires redhat-rpm-config
+%else
+%latest_requires openEuler-rpm-config
```

Comment:
Instead of the ifdef, can't we simply require `system-rpm-config`?  On RHEL and Fedora:
```
$ rpm -q --whatprovides system-rpm-config
redhat-rpm-config-252-1.fc38.noarch
```

Reply:
Sure thing!
```
[lcr@lcrpc ~]$ rpm -q --provides openEuler-rpm-config|grep system-rpm-config
system-rpm-config = 30-35.oe2303
```